packages/apps/plugins/plugin-status-bar/src/components/StatusBar.tsx

```diff
@@ -27,7 +27,10 @@ const StatusBarButton = forwardRef<HTMLButtonElement, StatusBarButtonProps>(
       <Component
         role='button'
         className={mx(
-          'flex items-center gap-2 p-1 px-2 rounded-sm select-none hover:bg-gray-200 cursor-pointer active:bg-gray-300',
+          'flex items-center gap-2 p-1 px-2 rounded-sm',
+          'select-none cursor-pointer',
+          'hover:bg-neutral-75 active:bg-neutral-150',
+          'dark:hover:bg-neutral-750 dark:active:bg-neutral-700',
```

Comment:
We have the `surface-hover` token for this, see if that works for this case. Generally we haven’t bothered with CSS active states, not that it needs to be removed here necessarily – if you feel principled about it, it should be added to the system.

Reply:
It didn't look right in this case. I quite like active states, we should consider adding it to the system as you say.

Reply:
I’m wary, then, of introducing another semantic token for backgrounds applied only on hover purely for aesthetic reasons, and the exception begs questions… should the hover semantic token be adjusted? Do these physical colors work here and not elsewhere, and if so why?